kernel/tlb_shootdown/src/lib.rs

```diff
@@ -40,6 +40,7 @@ pub fn init() {
 pub fn handle_tlb_shootdown_ipi() -> bool {
     let pages_to_invalidate = TLB_SHOOTDOWN_IPI_PAGES.read().clone();
     if let Some(pages) = pages_to_invalidate {
+        // THIS LOG USAGE CAN CAUSE A DEADLOCK ON AARCH64
```

Comment:
it can cause a deadlock on x86 too, actually, since it's also executing in an NMI context.

```suggestion
        // Note: logging in a NMI (x86_64) or FIQ (aarch64) context can cause deadlock,
        // so this should only be used sparingly to help debug problems with TLB shootdowns.
```